src/subscription/SubscriptionPage.messages.js

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  'subscription.heading.page': {
```

Comment:
This is repeated from L57 on `SubscriptionPage.jsx`, you don't need the translation with `id` and `defaultMessage` in both the `messages.js` file and in the `<FormattedMessage>` component